plugins/modules/podman_image.py

```diff
@@ -424,20 +424,27 @@ def _get_id_from_output(self, lines, startswith=None, contains=None, split_on='
     def present(self):
         image = self.find_image()
 
+        if image:
+            digest_before = image[0]['Digest']
```

Comment:
Unfortunately podman inspect info is not consistent along versions, so in 1.8 it's everything lowercase - `"digest"`
I'd suggest to force lower case in inspection function `self.find_image()` so that we can use only `['digest']`
Or something like: 
```python
digest_before = image[0].get('Digest', image[0].get('digest'))
```

Reply:
Yes, I had noticed that myself while trying to use this earlier today! What a pain. I'll fix it up.